proposals/selfAssemblingModelHierarchy/README.md

```diff
@@ -0,0 +1,208 @@
+
+Let's look at an example of model hierarchy correctly used.
+```
+def Scope "TriStateArea" (kind = "group") {
+    def Xform "NewYorkCity" (kind = "assembly") {
+        def Scope "Water" (kind = "group") {}
+        def Scope "Bridges" (kind = "group") {
+            def Xform "BrooklynBridge" (kind = "component") {}
+        }
+        def Scope "Tunnels" (kind = "group") {
+            def Xform "LincolnTunnel" (kind = "component") {}
+        }
+        def Scope "Boroughs" (kind = "group") {
+            def Scope "Manhattan" (kind = "assembly") {
+                def Scope "FifthAvenue" (kind = "group") {
+                    def Xform "DepartmentStore" (kind = "component") {
+                        def Scope "GroundFloor" {
+                            def Scope "Housewears" (kind = "subcomponent") {
+                                def Xform "Aisle_1" {
+                                    def Mesh "TopShelf" {
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+> **Note** "GroundFloor" and "Aisle_1" do not have `kind` authored. It would be incorrect to set `kind=group` or `kind=assembly`
+here. As a descendant of `component`, it may only be unset, or a `subcomponent`.
+
+> **Note** There's no relationship between the type of a prim being a `Scope` or an `Xform` and its authored `kind`.
+
+We've described `assembly` prims as being "important" prims generally corresponding to referenced assets. While not provided or mandated by USD, it is likely that pipeline and tooling is supporting versioning and setup of `assembly` and `component` models.
+
+For prims with `kind=group`, importance in model hierarchy is implicit from the fact that they _may_ contain important prims. `group` tagging exists solely so that traversal knows to keep going.
+
+## Problem
+There aren't currently consequences to violating model hierarchy so it happens a lot. Model hierarchy doesn't affect rendering so tools and users may not be properly maintaining it. Even pipelines that attempt to honor model hierarchy may choose to repair model hierarchy only at specific validation points.
+
+What makes this worth addressing now is the new pattern based collection [proposal](https://github.com/PixarAnimationStudios/USD-proposals/pull/4) aims to leverage model hierarchy in its predicates. Model hierarchy will now affect (and potentially accelerate) collection membership computation. **Tagging a prim as an `assembly` or a `component` could change the results of material bindings or light linking.**
+
+Proper maintenance of model hierarchy involves performing hygene up and down the hierarchy. For users to rely on model hierarchy in their predicates, model hierarchy maintenance can't be deferred to a validation script or ignored.
```

Comment:
Would the resolution of this question be to replace the indicated section with a note about preferring correctness problems to speed problems, and that the new validation framework could or must validate MH?

Reply:
I'll update the proposal that better considers the implications of validation, but I'm not sure I'm drawing the same conclusion. If validation can be used to detect and repair correctness problems, it seems like it could be used to detect and report any performance problems related to how a model hierarchy is structured.